demo/nuxeo-file/index.html

```diff
@@ -39,6 +39,11 @@
           <div>[[upload.upload-batch]]</div>
         </nuxeo-demo-section>
 
+        <nuxeo-demo-section heading="File upload button required invalid" size="small">
```

Comment:
Missing `<nuxeo-connection url="http://localhost:8080/nuxeo"></nuxeo-connection>` before the first section, otherwise `nuxeo-file` will try to contact the server on a different port, resulting in an error.